ipaserver/plugins/trust.py

```diff
@@ -1224,7 +1236,7 @@ def get_dn(self, *keys, **kwargs):
             return _trustconfig_dn[kwargs['trust_type']]
         except KeyError:
             raise errors.ValidationError(name='trust_type',
-                error=_("unsupported trust type"))
+                                         error=_("unsupported trust type"))
```

Comment:
indention style, rather then indenting error, please move name to the next line and dedent it. Space is precious, you know?